src/plasmapy/utils/data/downloader.py

```diff
@@ -287,7 +287,10 @@ def _http_request(self, url: str) -> requests.Response:
         # Only send GitHub api authorization if querying GitHub
         # auth = self._api_auth if "github.com" in url else None
 
-        headers = {"Content-Type": "application/json"}
+        headers = {
+            "Content-Type": "application/json",
+            "User-Agent": "PlasmaPy.utils.Downloader",
```

Comment:
Without a User-Agent flag, GitHub didn't know who was sending the HTTP requests. A stack overflow post gave me the idea to try this, and it seems to be working now. This should distinguish us from other potential bad actors who might DOS the API (although I thought our token already did that...).

Reply:
Huh! Good job tracking this down!  

> other potential bad actors

That doesn't count us as potential bad actors, does it? 😅 (I was going to save my bad acting for my forthcoming science fiction musical: _The Brain Hamsters of Epsilon Eridani_. 🚀🧠🐹)

Reply:
Honestly I'm not sure whether github wants us to be using a repo like this or not? But we shouldn't be sending a very large number of requests or anything...and the files aren't particularly large?